City-of-Helsinki/helsinki-paatokset
# PP-317: Added social media share buttons

Ticket: https://helsinkisolutionoffice.atlassian.net/browse/PP-317

**How to test**

Checkout branch and run `drush cim -y; drush cr`

The social media share buttons should now be on pages: Policymaker, Case, Decision, Minutes/etc. Test that the buttons work (I couldnt test twitter since I don't have it). 

Check that on case/decision pages the url in the share button changes if changes are made in the dropdown
